core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -68,6 +223,130 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
         );
     });
 
+    readonly getPages = this.effect(
```

Comment:
This is called 2 times.

Reply:
![image](https://user-images.githubusercontent.com/37185433/218863262-2ce49067-2946-46d4-8950-f46d28bdbf0c.png)

after checking, on line 8 that function is being called twice, might be related that for some reason on the init the data is loaded and then when the data is rendered for an unknown reason (might be related with the scrollbar that triggers the lazyload) the data fetch is called again

Reply:
`p-table` call it on init... you can use that for the first load, right?

Reply:
@fmontes , the `onLazyLoad` method is called automatically by `p-table` based on the scroll event, I don't think it's something you can control